spice/spice.py

```diff
@@ -44,15 +44,16 @@ class SpiceResponse:
     """
 
     input_tokens: int
-    """The number of input tokens given in this response."""
+    """The number of input tokens given in this response. If response is interrupted, may not be exact."""
```

Comment:
may be more clear to specify it may not be exact if the response is a partial streaming response, rather than final response